docs/modules/services/pages/videoconferences.md

```diff
@@ -0,0 +1,40 @@
+# Videoconferences
+
+For embedded videoconferences in the Meetings component, some options can be defined to use a server and api other than the default one.
+
+## Jitsi
+
+### Integration
+
+Set the domain and API URL for your Jitsi server in
+`config/initializers/decidim.rb` and `config/secrets.yml`
+
+An example snippet in `config/initializers/decidim.rb` may be:
+
+```ruby
+config.videoconferences = {
+  jitsi: {
+    domain: Rails.application.secrets.videoconferences[:jitsi][:domain],
+    api_url: Rails.application.secrets.videoconferences[:jitsi][:api_url]
+  }
+}
+```
+
+and then in `config/secrets.yml`:
+
+```yaml
+  videoconferences:
+    jitsi:
+      domain: <%= ENV["JITSI_DOMAIN"] %>
+      api_url: <%= ENV["JITSI_API_URL"] %>
+```
+
```

Comment:
Could we add here some examples? I'd write here the official URLs of jitsi, https:///meet.jit.si and ? (I don't actually know the API url 😅)